### PR TITLE
add memcache

### PR DESCRIPTION
**Pull request by keithblaha**
cookie cuts you some memcache

### CODE REVIEW

**Comment by keithblaha** [comment]
:eyeglasses: @spladug

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/__init__.py [hunk 1]
```diff
@@ -77,6 +80,10 @@ def make_wsgi_app(app_config):
     baseplate.add_to_context("events_production", EventQueue("production"))
     baseplate.add_to_context("events_test", EventQueue("test"))
 {% endif -%}
+{%- if cookiecutter.integrations.memcache %}
+    memcache_pool = memcache_pool_from_config(app_config, prefix="memcache.")
+    baseplate.add_to_context("memcache", MemcacheContextFactory(memcache_pool))
+{% endif -%}
 {%- if cookiecutter.integrations.redis %}
     redis_pool = pool_from_config(app_config, prefix="redis.")
```
Comment (keithblaha) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/__init__.py]: should i rename this redis_pool_from_config like i did for memcache?

Comment (spladug) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/__init__.py]: good point about the name conflict. how about this instead:

the imports for all the integrations become this style:

```python
from baseplate.context import memcache
```

then we can do

```python
memcache_pool = memcache.pool_from_config(...)
```

Comment (keithblaha) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/__init__.py]: k, i changed them both to work this way

**Comment by keithblaha** [comment]
:haircut_woman:

**Comment by keithblaha** [comment]
fyi i dont have push permissions- feel free to merge

**Comment by keithblaha** [comment]
no urgency, just wondering if you saw the above

**Comment by spladug** [comment]
no, i hadn't. thanks for the reminder.

merged as 91e8c86a8c896908237183d9e28064c22b2774d6.

**Pull request closed without merging**